src/components/Select/CountrySelect.tsx

```diff
@@ -31,7 +31,11 @@ export interface CountrySelectProps<
   label?: string;
   onChange: (changed: ICountryOption | null) => void;
   styles?: StylesConfig<Option, IsMulti, Group>;
-  value: ICountryOption | null;
+  /**
+   * @example
+   * value={countryOptions.filter(x => x.value === countryCode)}
+   */
+  value: ICountryOption[] | ICountryOption | null;
```

Comment:
Since we have CountrySelect (single) and CountryMultiSelect (multiple), will this `CountrySelect` component accept a list of ICountryOption?